docs/handbook/spending-money.md

```diff
@@ -0,0 +1,74 @@
+# Spending money
+
+There are many occassions when you will need to spend company money.
+
+PostHog is a lean organization - the less we spend, the more time we have to make sure the company takes off. However, it is more important you are productive, healthy and happy.
+
+Please just spend company money like it's your own.
+
+If it's a trivial expense, just buy it. We'll provide you with a company card with a $500/month spending limit for this reason.
+
+If it's a larger expense and doesn't fit into the items here, please **raise a policy suggestion for it as a pull request** in this doc, so we can document our decision making into our policy rather than making everything case by case.
+
+## Trivial expenses
+
+Just do it.
+
+This means expenses that are under $75 one off or under $20/month recurring that we can cancel easily.
+
+Make sure you keep copies for all receipts. We *cannot* pay you back for anything without a receipt.
+
+## Making larger purchases
+
+If your purchase fits within the policy below, there is no need to ask. We **cannot** pay you back for anything without a receipt.
```

Comment:
I get _why_ you're saying this but this feels like a policy for a larger company where the overhead of getting some form of "approval" is very high. It feels like we need to be very careful with cash so I don't think it's too much to ask either me or you for a one-line yes/no.

Reply:
I've seen before that if you have this type of policy, it can create a "I'll just request top of the line thing X", then you spend your life trying to push back on requests, spending more than you want and no one ends up happy. I'd rather people feel "I need to be able to justify this" and make their own decisions. Maybe we should make the language a little tighter around what you can buy instead - ie if we've said a prefurbished Macbook pro, maybe we just set a clearer model or spend amount?

Reply:
Okay that's good context and I do prefer treating people like adults who can own decisions. i do think it's good to be very descriptive as otherwise one person is going to be very sensible and get a lower spec model and then someone else will get a super high spec model and there will be jelousy

Reply:
#26 created to solve this